modules/candidate_list/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate List
+
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
+
+## Intended Users
+
+The candidate list is primarily used by data entry staff to use in
+order to access the `timepoint_list` module (and `instrument_list`
+from there) in order to access the candidate and perform data entry.
+
+## Scope
+
+The `candidate_list` module provides a list of candidates, as well
+as linking to other relevant LORIS modules where applicable.
+
+## Permissions
+
+Accessing the `candidate_list` module requires either the `data_entry`
+permission and at least one study site, or the `access_all_profiles`
+permission.
+
+Users with the `access_all_profiles` permission can see every
+candidate in LORIS, while users with only the `data_entry` permission
+can only see candidates at study sites where they are affiliated.
+
+## Configurations
+
+The `useProjects` configuration affects whether the table has a
+"Project" column for the candidate, and whether there is a filter
+dropdown for that column.
+
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
+
+## Interactions with LORIS
+
+The "Scan Done" column indicates whether or not a scan was performed
+for that candidate. When a scan was performed (i.e. it has the value
+"Y"), clicking the table cell links to the `imaging_browser` module
+prefiltered for that candidate.
+
+Clicking on the cell in the PSCID column takes the user to the
+`timepoint_list` page for that candidate. The link is only clickable
+if the user does *not* have the `data_entry` permission. For users
+who have the `data_entry` permission, a separate form appears for
```

Comment:
is this behaviour 100% triple-checked?   I trust you if so, but I'm not confident it behaves this way.

Reply:
@driusan maybe check with @Jkat as I don't recall this is the case for PREVENT-AD. Maybe it is configurable or PREVENT-AD is overwriting the module. Both are possible.

Reply:
If this isn't the behaviour, it's a bug in the code that should be fixed.